### PR TITLE
cmd/benchcore: raise open file limit

### PR DESCRIPTION
**Pull request by jbowens**
We hit the open file limit if we try to do > 1000 txs/sec. Raise the limit on the cored machine.

### CODE REVIEW

**Comment by bobg** [comment]
This PR has no description, and the title belies the biggest change here, which is to run `cored` under `sudo`. Is it possible to avoid that? If not, please document it.

**Comment by jbowens** [comment]
This a benchmark, not a production server ;)

It is likely possible, but I don't know how. @kr and I paired on this and he might have an idea.

**Comment by ryandotsmith** [comment]
I bet we only need `sudo` for ulimit.

**Comment by jbowens** [comment]
I think ulimit only applies to the current shell, so as soon as we exit the inner shell,  we lose it. We might need to exec something as the original user.

**Comment by kr** [comment]
Good call @bobg, we should switch back to an unprivileged user after the ulimit. Maybe like

```
sudo bash <<EOFROOT
ulimit -n 65535

sudo -u ubuntu bash <<EOFUBUNTU
export DATABASE_URL='{{dbURL}}'
export MAXDBCONNS=100
export GOTRACEBACK=crash
./cored 2>&1 | tee -a cored.log
EOFUBUNTU

EOFROOT
```

**Comment by erykwalder** [comment]
Another option to one-line it:
`sudo bash -c "ulimit -n 65535 && exec su ubuntu"`

File: cmd/benchcore/main.go [hunk 1]
```diff
@@ -765,6 +765,7 @@ $HOME/corectl create-token -net benchcorenet > $HOME/network-token.txt
 
 const coredsh = `#!/bin/bash
 set -eo pipefail
+sudo bash -c "ulimit -n 65535 && exec su ubuntu"
```
Comment (kr) [inline cmd/benchcore/main.go]: This line doesn't do anything useful. It starts a bash process, which changes its own ulimit, then execs a new process as a different user, which does nothing and then exits.

**Comment by jbowens** [comment]
PTAL

**Comment by kr** [comment]
LGTM

**Pull request merged**